Client/Frontend/Fakespot/Views/FakespotErrorCardView.swift

```diff
@@ -117,9 +121,15 @@ final class FakespotErrorCardView: UIView, ThemeApplicable {
         cardView.configure(cardModel)
     }
 
+    private var starRatingHeightConstraint: NSLayoutConstraint?
```

Comment:
Copy & paste mistake? 🤔 I don't think the error card has a star rating so we probably name this variable differently to avoid confusion.